provision/testdata/help.stdout.txt

```diff
@@ -3,7 +3,7 @@ usage: acc_provision.py [-h] [-v] [--release] [--debug] [--sample] [-c file]
                         [-p pass] [-w timeout] [--list-flavors] [-f flavor]
                         [-t token] [--test-data-out file] [--skip-kafka-certs]
                         [--upgrade] [--disable-multus disable_multus]
-                        [-s file]
+                        [-s file] [--apic-oobm-ip ip]
```

Comment:
@snaiksat Does this parameter name look okay ?

Reply:
thanks for checking @pkharat , should be okay